### ISSUE TITLE
Container: example(s) for long-running operations

### ISSUE THREAD

**Issue by DazWilkin** [GoogleCloudPlatform/google-cloud-python#5780]
Please provide (reference) an example for converting `container_v1.types.Operation` into `api_core.operatoin.Operation`.

[`create_cluster`](https://googlecloudplatform.github.io/google-cloud-python/latest/_modules/google/cloud/container_v1/gapic/cluster_manager_client.html#ClusterManagerClient.create_cluster) (and other methods in the Cloud Client Library) return (long-running) `Operation`(s).

I assumed (hoped) this would be `google.api_core.operation.Operation` that I could `operation.result()` but, the method turns [`google.cloud.container_v1.types.Operation`](https://googlecloudplatform.github.io/google-cloud-python/latest/container/gapic/v1/types.html#google.cloud.container_v1.types.Operation)

[Spanner's Python Library, for example, [returns](https://googlecloudplatform.github.io/google-cloud-python/latest/spanner/instance-api.html#google.cloud.spanner_v1.instance.Instance.create) google.api_core.operation.Operation in this case]

I'm struggling to connect this with (presumably?) [Long-Running Operations](https://googlecloudplatform.github.io/google-cloud-python/latest/core/operation.html) documented in Core.

I'm unable to new an `OperationsClient`. The Example suggests `operations_v1.OperationsClient()` but this errors with `TypeError: __init__() takes at least 2 arguments (1 given).`

It's unclear what `channel (grpc.Channel)` I should provide.

Is this the correct approach? Having triggered the cluster creation, I'd like to track its progress to completion.

**Comment by lukesneeringer** [comment]
Yep, we should add that and then this just magically works for @DazWilkin. What API is it?

**Comment by theacodes** [comment]
Container

**Comment by DazWilkin** [comment]
Ooh.... thanks @theacodes @lukesneeringer

**Comment by tseaver** [comment]
@lukesneeringer Ping?

**Comment by tseaver** [comment]
@lukesneeringer PR #6084 will automate the codegen for `google-cloud-container`, but somebody still needs to make the change to the upstream GAPIC config to get the operations wrapped.
